### DISCUSSION TITLE
Business proposal

### DISCUSSION THREAD

**Discussion by abbasovartem812** [socketio/socket.io#5138] [Q&A] [answered]
Hello,
We, BAM Studio, specialize in WOW videocontent (VFX, CGI) for various social networks, DOOH ads, websites and platforms. We are happy to offer our services to cryptocurrency companies looking to stand out in the highly competitive blockchain market through a new kind of digital marketing.

Our portfolio is here  Instagram: bam.cgi

Our Services

We provide a full range of high-quality VFX/CGI content creation services, including:

1. Visual Effects (VFX)
- Animated and promotional videos with 3D animation:
- Impressive product/service trailers

2. Computer Generated Imagery (CGI) 
- 3D models and visualizations
- Virtual worlds and avatars for metaverse and VR
- Character animation
- 3D visualization of blockchain concepts

We also create visuals for user, employees training, social media, digital advertising, posters, conferences and more.

Why Choose Us?

- High quality work by experienced CG specialists (Balenciaga, i61, financial banks, aviacompanies)
- Personalized concepts tailored to your needs
- Extensive experience in blockchain/crypto projects

We are ready to help you create unforgettable visuals that will showcase your innovations and increase your competitiveness.

Please contact us at bamcgi.studio@gmail.com or @bamcgi to discuss our proposal further.

**Comment by darrachequesne** [comment, marked as answer]
Hi!

Thanks for reaching out. We do not need your services right now, but we will not hesitate to contact you if we do in the future.

Have a great day!
